src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php

```diff
@@ -170,9 +170,11 @@ public function testGlobalsWithDifferentTypesAndValues()
 
         $calls = $container->getDefinition('twig')->getMethodCalls();
         foreach (array_slice($calls, 1) as $call) {
-            list($name, $value) = each($globals);
+            list($name, $value) = array(key($globals), current($globals));
```

Comment:
IMO you can just remove the whole line and use `key` and `current` in place of the variables directly.